db/migrate/20161021104336_create_authorized_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAuthorizedUser < ActiveRecord::Migration
```

Comment:
Fixed all the issues except the controller name issue as the comments suggested. Too lazy to reply other comments.